Without this patch, projects on Tiger will have an error like this
during the destroot stage:

dyld: Library not loaded: @loader_path/libgeocode-glib.0.dylib

--- gtkdoc/scangobj.py.orig	2021-09-11 13:08:06.000000000 -0400
+++ gtkdoc/scangobj.py	2021-09-11 13:09:52.000000000 -0400
@@ -1307,6 +1307,7 @@
         return res
 
     run_env = os.environ.copy()
+    run_env['DYLD_LIBRARY_PATH'] = options.module
     run_env['LC_MESSAGES'] = 'C'
     run_env.pop('LC_ALL', None)
     res = execute_command(options, 'Running',
